src/project/src/main/java/Pet.java

```diff
@@ -0,0 +1,44 @@
+public class Pet {
+    private String name;
+    private int age;
+    private String species;
+    private Person owner;
+
+    public Pet(String name, int age, String species) {
+        this.name = name;
+        this.age = age;
+        this.species = species;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
```

Comment:
setName, setAge, setSpecies heb je in principe niet nodig gezien deze eenmalig door de constructor gevuld worden en daarna liever niet meer overschrijft.